### DISCUSSION TITLE
docker deployment doris error

### DISCUSSION THREAD

**Discussion by zan-liu** [apache/doris#25869] [A - General / Q&A]
I tried to use docker to pull the official image and deploy doris, but FE encountered this error when starting up. Can you tell me how to fix it?
![image](https://github.com/apache/doris/assets/117903714/c111f6f6-5934-4af1-8b1b-d4a2cecf7de7)

os:mac
type:1FE 1BE
docker-componse:
`version: "3"
services:
  fe:
    image: apache/doris:2.0.0_alpha-fe-x86_64
    hostname: fe
    environment:
     - FE_SERVERS=fe1:192.168.1.167:9010
     - FE_ID=1
    volumes:
      - /tmp/1025/data/fe/doris-meta/:/opt/apache-doris/fe/doris-meta/
      - /tmp/1025/data/fe/log/:/opt/apache-doris/fe/log/
    network_mode: host
  be:
    image: apache/doris:2.0.0_alpha-be-x86_64
    hostname: be
    environment:
     - FE_SERVERS=fe1:192.168.1.167:9010
     - BE_ADDR=192.168.1.167:9050
    volumes:
     - /tmp/1025/data/be/storage/:/opt/apache-doris/be/storage/
     - /tmp/1025/data/be/script/:/docker-entrypoint-initdb.d/
    depends_on:
      - fe
    network_mode: host`

**Comment by ixzc** [comment]
show your compose.yaml

**Reply by zan-liu** [reply]
this is my compose.yaml

version: "3"
services:
   fe:
     image: apache/doris:2.0.0_alpha-fe-x86_64
     hostname: fe
     environment:
      - FE_SERVERS=fe1:192.168.1.167:9010
      - FE_ID=1
     volumes:
      - ./fe/doris-meta/:/opt/apache-doris/fe/doris-meta/
      - ./fe/log/:/opt/apache-doris/fe/log/
     network_mode: host
   be:
     image: apache/doris:2.0.0_alpha-be-x86_64
     hostname: be
     environment:
      - FE_SERVERS=fe1:192.168.1.167:9010
      - BE_ADDR=192.168.1.167:9050
     volumes:
      - ./be/storage/:/opt/apache-doris/be/storage/
      - ./be/script/:/docker-entrypoint-initdb.d/
     depends_on:
       - fe
     network_mode: host

**Reply by zan-liu** [reply]
![image](https://github.com/apache/doris/assets/117903714/74ef7c43-9a53-4532-b11a-c7ae2f4bdd61)

**Comment by zan-liu** [comment]
@ixzc If you have time, please help me, thank you very much!!!

**Reply by ixzc** [reply]
You can create a network named doris_net with a subnet of 192.168.1.0/24：
docker network create --subnet=192.168.1.0/24 doris_net 

and use this compose :

version: "3"
services:
fe:
image: apache/doris:2.0.0_alpha-fe-x86_64
hostname: fe
environment:
- FE_SERVERS=fe1:192.168.1.2:9010
- FE_ID=1
volumes:
- ./fe/doris-meta/:/opt/apache-doris/fe/doris-meta/
- ./fe/log/:/opt/apache-doris/fe/log/
network_mode: host
be:
image: apache/doris:2.0.0_alpha-be-x86_64
hostname: be
environment:
- FE_SERVERS=fe1:192.168.1.2:9010
- BE_ADDR=192.168.1.2:9050
volumes:
- ./be/storage/:/opt/apache-doris/be/storage/
- ./be/script/:/docker-entrypoint-initdb.d/
depends_on:
- fe
network_mode: doris_net
